src/main/java/org/wildfly/security/auth/client/AuthenticationConfiguration.java

```diff
@@ -833,6 +835,17 @@ public AuthenticationConfiguration useKeyManagerCredential(X509KeyManager keyMan
     }
 
     /**
+     * Create a new configuration which is the same as this configuration, but which uses local kerberos ticket cache
+     * to acquire the credential required for authentication.
+     *
+     * @param mechanismOids array of oid's indicating the mechanisms over which the credential is to be acquired
+     * @return the new configuration
+     */
+    public AuthenticationConfiguration useLocalKerberosCredential(Oid[] mechanismOids) {
```

Comment:
The master branch of WildFly Elytron is heading to a 1.2.0 release, I think any new API methods should also add the since annotation with a value of 1.2.0